Cartfile

```diff
@@ -2,7 +2,7 @@ binary "https://www.mapbox.com/ios-sdk/MapboxAccounts.json" ~> 2.3.0
 binary "https://api.mapbox.com/downloads/v2/carthage/mobile-navigation-native/MapboxNavigationNative.json" ~> 18.0.2
 binary "https://api.mapbox.com/downloads/v2/carthage/mobile-maps/mapbox-ios-sdk-dynamic.json" == 6.1.0-beta.2
 binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon-ios.json" == 4.0.0
-github "mapbox/mapbox-directions-swift" ~> 0.33
+github "mapbox/mapbox-directions-swift" "2284-route-refresh-directions"
```

Comment:
This is temporary. We’ll release v0.34.0 before or right after merging #2366 so that the CocoaPods integration tests pass.

Reply:
We need to also update the CocoaPods podspecs and CocoaPods integration tests once v1.0.0-rc.1 is pushed to CocoaPods trunk.